bin/singa-run.sh

```diff
@@ -33,7 +33,8 @@ usage="Usage: singa-run.sh -conf <job config file> [ other arguments ]\n
 
 # parse arguments
 # make sure we have '-conf' and remove '-exec'
-exe=./singa
+#exe=./singa
+exe=./src/singa.py
```

Comment:
no need to change the code here.
we should be compatible with the non-python version.
./bin/singa-run.sh will call ./singa by default. 
we can add `-exec tool/python/singa.py` to run the python version.